src/components/show-products/index.js

```diff
@@ -43,19 +67,34 @@ export const ShowProducts = () => {
 
   return (
     <ContentContainer>
+      <form className="filter-form">
+        <label htmlFor="filter">
+          Filter items
+          <input
+            className="input-item"
+            type="search"
+            id="filter"
+            name="q"
```

Comment:
I have a question, what does this mean `name="q"`? :eye: :eye:
Great use o the type="search" :100: :1st_place_medal:

Reply:
This detail went unnoticed, let's fix it

Reply:
I also loved the use of search ...